prov/verbs/src/fi_verbs.h

```diff
@@ -789,4 +792,93 @@ fi_ibv_send_iov_flags(struct fi_ibv_ep *ep, struct ibv_send_wr *wr,
 	return fi_ibv_send_poll_cq_if_needed(ep, wr);
 }
 
+static inline const void *
+fi_ibv_dgram_moderate_user_buf(const void *user_buf, size_t offset)
```

Comment:
What does the term 'moderate' mean with respect to this set of calls?

Reply:
I'm not sure that's the term 'moderate' correct here. Do you have any suggestion?
I mean that the functions change user's buffer/iov to consider message prefix that's provided in the message (ibv doesn't require this for the send message -- that's why we rewind the user's buffer/iov --, it is only required for receive buffer)